src/ecs/systems/mod.rs

```diff
@@ -1,19 +1,23 @@
 //! Built-in `specs` `System`s.
 
+mod audio;
+mod dj;
+mod rendering;
+mod transform;
+
+pub use self::audio::AudioSystem;
+pub use self::dj::DjSystem;
 pub use self::rendering::RenderSystem;
 pub use self::transform::TransformSystem;
 
 // use config::Config;
 use error::Result;
 use ecs::{System, World};
 
-mod rendering;
```

Comment:
Actually, the standard seems to be to put `use` before `mod`, although I did it the other way around before I read about it.